contracts/LifMarketMaker.sol

```diff
@@ -197,7 +210,9 @@ contract LifMarketMaker is Ownable {
 
   // Called from the foundation wallet to claim eth back from the Market Maker. Maximum amount
   // that can be claimed is determined by getMaxClaimableWeiAmount
-  function claimEth(uint256 weiAmount) {
+
```

Comment:
why this newline? Let's be consistent on this kind of things, after the comment before the function there's usually no newline, or is it? I also don't like the initial newline in the function body but I can live with it